ResearchKit/Consent/ORKConsentSignatureFormatter.h

```diff
@@ -0,0 +1,8 @@
+#import <ResearchKit/ResearchKit.h>
+#import <ResearchKit/ORKConsentSignature.h>
+
+@interface ORKConsentSignatureFormatter : NSObject
+
+- (NSString *)HTMLForSignature:(ORKConsentSignature *)signature;
```

Comment:
A little unfortunate to have to allocate an object just to call this stateless method. That said, it does make the test code considerably cleaner than the alternatives I've thought of.